.github/workflows/cryostat-IT.yml

```diff
@@ -0,0 +1,121 @@
+on:
+  workflow_call:
+    inputs:
+      build-arch:
+        required: true
+        type: string
+      skip-itests:
+        required: false
+        type: boolean
+        default: false
+    outputs:
+      image-version:
+        description: the Cryostat application version that will be built
+        value: ${{ jobs.get-pom-properties.outputs.image-version }}
+
+jobs:
+  get-pom-properties:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: ${{ github.repository_owner }}/cryostat
+        ref: main
+    - uses: Mudlet/xmlstarlet-action@master
+      with:
+        args: ed -P -L -u '//_:properties[_:io.cryostat.core.version="2.23.0"]/_:io.cryostat.core.version' -v "2.24.0-SNAPSHOT" pom.xml 
```

Comment:
Can we remove `2.23.0` and pass in `2.24.0-SNAPSHOT` as workflow argument?

This way, we don't have to go update these versions every release.